README.md

```diff
@@ -36,31 +36,40 @@ You will want to set up an IDE before you begin making contributions to this cod
 
 A download of VS Code is available [here](https://code.visualstudio.com/download). Once you have installed the editor and you have the program open you can open the command pallet (shift + command + P) and search for `install 'code' command in PATH`.  Once you run that command from the command pallet, you can type `code <directory name>` to open any directory in VS Code from the command line.
 
-## Cloning the repository
+### Cloning the repository
 
 Navigate to a repository where you would like to store the source code.  Then run
 
 ```bash
 git clone https://github.com/codeforportland/symptomsurvey_frontend.git
 code symptomsurvey_frontend
 ```
-## Installing the Google Maps API secret key
 
-In order for the app to make API calls to Google Maps, which places the Twitter markers, you will need a secret key installed on your machine. This is done in the terminal, it's not kept in any files so there's no risk of uploading it accidentally. **You will need to talk to someone on the frontend team to get the key directly from them.** Follow these steps (for Mac OS):
+### Environment Variables
 
+There is one environment variable that is required for access to Google Maps, G_MAPS_KEY, which places the Twitter markers. An easy platform-agnostic way to ensure the variables are set appropriately is to add a .env file at the project root with the variables defined on separate lines like so: G_MAPS_KEY=value. The actual value can be provided by the frontend team.
+
+### Installing the Google Maps API secret key
+
+Follow these steps for Mac OS:
+1. Obtain key from frontend team
+2. $`echo "export G_MAPS_KEY=<gmaps key>;" >> ~/.bash_profile`
+3. $`. ~/.bash_profile` or restart the project
+
+Follow these steps for Windows OS:
 1. Obtain key from frontend team
-1. $`echo "export G_MAPS_KEY=<your gmaps key>;" >> ~/.bash_profile`
-1. $`. ~/.bash_profile` or restart the project
+2. In the terminal $`echo G_MAPS_KEY=<gmaps key> >> .env`
+3. Restart the project
```

Comment:
Slightly misleading, the .env solution works for any *nix based runtime so the MacOs users can use this solution as well. Can you change this up so that it says both solutions would work for all cases? Gitbash doesn't come with a .bash_profile by default but you can easily make one using the same steps as above and it will use it for its local settings.

Reply:
@AlexanderNull I didn't want to touch that part of it because I'm not familiar with Linux/Unix OS but since we will be using .env in both I'll go ahead and update it.